.circleci/config.yml

```diff
@@ -9,6 +9,26 @@ references:
     rubygem: &RUBYGEM_CACHE_KEY static-site-gems-v1-{{ checksum "content/Gemfile.lock" }}
 
 jobs:
+  website-test:
+    machine:
+      image: ubuntu-1604:201903-01 
+    steps:
+      - checkout
+
+      - run:
+          name: gem and bundle install 
+          command: |
+            gem install bundler -v '1.17.3' --no-document
+            bundle _1.17.3_ install --jobs=3 --retry=3
+
+      - run:
+          name: make sync
```

Comment:
The name will be the command if you don't set this. So doing `run: make sync` will give you the same result.

Reply:
thanks alvin!